src/patternfly/components/Breadcrumb/breadcrumb-item-divider.hbs

```diff
@@ -0,0 +1,3 @@
+<span class="pf-c-breadcrumb__item-divider">
+  <i class="fas fa-angle-right"></i>
+</span>
```

Comment:
Thanks for adding an element in for the icon. Can you please add `aria-hidden="true"`?

Reply:
`aria-hidden="true"` gets rendered on the icon. Do you want me to add it to `pf-c-breadcrumb__item-divider` as well?

Reply:
I have mixed feelings about this. 

I realize that font awesome icons automatically apply attributes for a11y. But my reasons for wanting to include `aria-hidden` anyway are:
- In case we opt for a different icon solution that doesn't do this. 
- I feel like the HTML we present for these should serve as a pattern that represents what we recommend as best practice, and serves as a way of bringing more awareness to these things. Since this attribute is added dynamically, it could be easily overlooked as something that's important. If we provide sample html that could be copied and pasted but is incomplete, we're potentially contributing bad code to the world.

I could easily be convinced otherwise. Maybe it would be sufficient to just include an a11y documentation comment or table row that mentions this attribute?

Reply:
Wait, I'm just now seeing the `aria-hidden`. Was it there the whole time?!?

Yes, this is find.